projects/igniteui-angular/src/lib/grids/toolbar/grid-toolbar.component.html

```diff
@@ -1,90 +1,29 @@
-<span class="igx-grid-toolbar__title" *ngIf="getTitle()">
-    {{ getTitle() }}
+<span class="igx-grid-toolbar__title">
+    <ng-template #title>
+        <ng-content select="[igxGridToolbarTitle],igx-grid-toolbar-title"></ng-content>
+    </ng-template>
+    <igx-grid-toolbar-title *ngIf="!hasTitle">Toolbar</igx-grid-toolbar-title>
+    <ng-container *ngTemplateOutlet="title"></ng-container>
 </span>
 
-<div class="igx-grid-toolbar__custom-content" *ngIf="customContentTemplate != null">
-    <ng-container *ngTemplateOutlet="customContentTemplate; context: context">
-    </ng-container>
+<div class="igx-grid-toolbar__custom-content">
+    <ng-content></ng-content>
 </div>
 
 <div class="igx-grid-toolbar__actions">
```

Comment:
Can we perhaps remove this div and move the class to the igx-grid-toolbar actions element?